exadata-database-service-on-cc/introduction/introduction.md

```diff
@@ -0,0 +1,92 @@
+This network is fully contained within the Exadata Cloud@Customer rack and does not connect to your corporate network.
+However, the Exadata infrastructure is indirectly connected to your corporate network through the control plane servers to obtain Domain Name System (DNS) and Network Time Protocol (NTP) services to the Exadata Cloud infrastructure.
+Therefore, the IP addresses that are allocated to the administration network must not exist elsewhere in your corporate network.
+
+Oracle requires that you register the host names and IP addresses for the Exadata Cloud@Customer client and backup network interfaces in your corporate DNS.
+As part of the deployment process, you will define the host names and IP addresses to be used for the various Exadata Cloud@Customer network interfaces.  
+The Exadata Cloud@Customer deployment requires that at least one reliable DNS server be available and that it must be accessible to the control plane servers and to all of the servers on the client network.
+You can register up to three DNS servers as part of the Exadata Cloud@Customer deployment to ensure coverage in case a server becomes unavailable.
+
+Exadata Cloud@Customer uses your NTP service to ensure that all of the system components are synchronized to the same time.
+The deployment will require that at least one reliable NTP server is available and is accessible to the control plane servers and to all of the servers on the client network.
+You can register up to three NTP servers as part of the Exadata Cloud@Customer deployment to ensure coverage in case a server becomes unavailable.
+
+The Oracle Cloud Operations team can use the OCI Console or REST APIs over an HTTPS or SSH connection to manage the infrastructure.
+Their HTTPS connections travels to the infrastructure through the Oracle service tenancy via the admin VCN, and then through the persistent, secure automation tunnel.
+
+
+These hands-on lab guides provide step-by-step directions to setting up and using your Exadata Database Service on Cloud@Customer
+
+
+## Lab Breakdown
+
+* Lab 1. Create Infrastructure Resource for Exadata Database Service on Cloud@Customer
+* Lab 2. Create a VM Cluster Network
+* Lab 3. Create a VM Cluster Resource for Exadata Database Service on Cloud@Customer
+* Lab 4. Create an Oracle Database on Exadata Database Service     
+
+
+
+
+
+## Learn More
+
+* Click [here](https://docs.oracle.com/en/engineered-systems/exadata-cloud-at-customer/ecccm/ecc-whats-new-in-exadata-cloud-at-customer-gen2.html#GUID-2F27E5A9-67A8-459E-9453-81E1F68043DE) to learn about What's New in Exadata Database Service on Cloud@Customer.
+
+## Acknowledgements
+
+* **Author** - Leo Alvarado, Eddie Ambler, Product Management
+
+* **Contributors** - Tammy Bednar, Product Management
+
+* **Last Updated By** - Leo Alvarado, Product Management, April 2023.
+
+**You are all set. Let us begin!**
```

Comment:
@leoneila - delete the You are all set line. Instead you can add You may now **proceed to the next lab** before Learn more section.